package.json

```diff
@@ -49,20 +47,21 @@
     "lint-staged": "^15.2.10"
   },
   "resolutions": {
-    "**/nomnom/underscore": "^1.12.1",
-    "**/core/ember-inline-svg/**/nth-check": "^2.0.1",
-    "**/ember-cli/**/ansi-html": "^0.0.9",
-    "**/ember-cli/testem/fireworm/async": "^2.6.4",
-    "**/@hashicorp/design-system-components/ember-stargate": "^0.6.0",
-    "micromatch": "^4.0.8"
+    "nomnom/underscore": "^1.12.1",
+    "@hashicorp/design-system-components/ember-stargate": "^0.6.0",
+    "micromatch": "^4.0.8",
+    "ember-inline-svg/nth-check": "^2.0.1",
```

Comment:
apologies im barely noticing that `nth-check` isn't a direct dependency of `ember-inline-svg`. im thinking `"css-select/nth-check": "^2.0.1",` should work

Reply:
oops good catch, yes it "css-select" when I do yarn why, changing it